lib/msal-browser/docs/v2-migration.md

```diff
@@ -0,0 +1,60 @@
+# Migrating from MSAL 2.x to MSAL 3.x
+
+If you are new to MSAL, you should start [here](initialization.md).
+
+If you are coming from [MSAL v1.x](../../msal-common/), you should check [this guide](v1-migration.md) first to migrate to [MSAL v2.x](../../msal-browser/).
+
+If you are coming from [MSAL v2.x](../../msal-browser/), you can follow this guide to update your code to use [MSAL v3.x](../../msal-browser/).
+
+## Update your code
+
+In MSAL v2.x, you created an application instance as below:
+
+```javascript
+import * as msal from "@azure/msal-browser";
+
+const msalConfig = {
+    auth: {
+        clientId: 'your_client_id'
+    }
+};
+
+const msalInstance = new msal.PublicClientApplication(msalConfig);
+```
+
+In MSAL v3.x, you should initialize the application object as well. There are several options at your disposal:
```

Comment:
Nit: I think we need to make it clear it's mandatory.
```suggestion
In MSAL v3.x, you must initialize the application object as well. There are several options at your disposal:
```

Reply:
Fixed